cache/src/main/kotlin/com/dropbox/android/external/cache4/Cache.kt

```diff
@@ -153,20 +160,9 @@ internal class CacheBuilderImpl : Cache.Builder {
         this.clock = clock
     }
 
+    @ExperimentalTime
```

Comment:
Since consumer will be forced to opt in to `@ExperimentalTime` whether they specify `expireAfterAccess ` / `expireAfterWrite` or not, let's move `@ExperimentalTime` to `interface Cache` and `internal class CacheBuilderImpl` so we can get rid of these on functions.